app/models/constants/involved_civilian.rb

```diff
@@ -55,12 +57,15 @@ module InvolvedCivilian
       perceived_armed: "Perceived armed?",
       confirmed_armed: "Confirmed armed?",
       resisted: "Resisted?",
-      received_force: "Force used by an officer?"
+      received_force: "Force used by an officer?",
+      k12_type: "Civilian's role on K-12 campus"
```

Comment:
Err, isn't this line still important for making the description on the review page look nicer, @everett-wetchler ?